sanguk/app.js

```diff
@@ -33,20 +33,77 @@ app.get("/ping", (req,res)=>{
 })
 
 app.post("/users/signup", async (req,res,next)=>{
-    const {username, password, email, phonenumber } =req.body
+    const {username, userProfileImage } =req.body
 
     await myDataSource.query(
         `INSERT INTO users(
             username,
-            password,
-            email,
-            phonenumber
-        ) VALUES(?, ?, ?, ?)`
-        ,[username, password, email, phonenumber]
+            userProfileImage
+        ) VALUES(?, ?)`
+        ,[username, userProfileImage]
     )
     res.status(201).json({message : "userCreated"})
 })
 
+app.post("/posts/signup", async (req,res,next)=>{
```

Comment:
```suggestion
app.post("/posts", async (req,res,next)=>{
```
signup 이라는 것은 말그대로 user 회원가입을 의미하며 사용하는 단어입니다. `/posts/signup`처럼 뒤에 signup이 붙을 필요는 없어보이고 http method 중 하나인 post 자체가 서버로 데이터를 전송하는 행위를 나타내기때문에 `app.post("/posts", . . .`라고 작성하셔도 충분할 것 같습니다 :)

Reply:
게시물 등록이라는 의미로 생각 하였는데 ,  이 부분은 자주 코딩 하면서 수정해볼게요^^